packages/cra-template-seed/template/src/pages/Fallback/Buggy.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/* eslint-disable no-unused-vars */
+class Buggy extends React.Component {
```

Comment:
what is this?

Reply:
You mean the component? or the `Welcome` string?

Reply:
The whole component, looks like this component is explicitly there for throwing errors?

Reply:
Its there for handling syntax error in the app.
Instead of receiving a white page, it prints this component.